chainer/functions/evaluation/accuracy.py

```diff
@@ -45,10 +46,16 @@ def forward(self, inputs):
             count = (pred == t).sum() - ignore_cnt
             total = t.size - ignore_cnt
 
-            if total == 0:
-                return xp.asarray(0.0, dtype=y.dtype),
+            if xp is numpy:
+                # Avoid warning of `divide by zero`
+                if total == 0:
+                    return xp.asarray(0.0, dtype=y.dtype),
+                else:
+                    return xp.asarray(float(count) / total, dtype=y.dtype),
             else:
-                return xp.asarray(float(count) / total, dtype=y.dtype),
+                return xp.where(total == 0,
+                                xp.asarray(0.0, dtype=y.dtype),
+                                xp.asarray(count / total, dtype=y.dtype)),
```

Comment:
py2 division problem?

Reply:
Thank you for pointing out. Fixed in 564f38825acc05453cdbf35e7fa7fe24578de293.